code/modules/mob/living/carbon/human/npc/orc.dm

```diff
@@ -48,6 +48,7 @@
 	job = "Ambush Orc"
 	ADD_TRAIT(src, TRAIT_NOMOOD, TRAIT_GENERIC)
 	ADD_TRAIT(src, TRAIT_NOHUNGER, TRAIT_GENERIC)
+	ADD_TRAIT(src, TRAIT_CRITICAL_RESISTANCE, TRAIT_GENERIC)//they are meant to be war tools like the Aasimars except being inhumen abominations
```

Comment:
```suggestion
```
I did say it's fair for them to not get critical weakness, but I never mentioned it's fine to give full crit resistance. No.